ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureBlobDataAccessor.java

```diff
@@ -237,6 +237,22 @@ public boolean downloadFile(String containerName, String fileName, OutputStream
     }
   }
 
+  /**
+   * Delete a file from blob storage, if it exists.
+   * @param containerName name of the container containing file to delete.
+   * @param fileName name of the file to delete.
+   * @return true if the file was deleted, otherwise false.
+   * @throws BlobStorageException for any error on ABS side.
+   */
+  boolean deleteFile(String containerName, String fileName) throws BlobStorageException {
```

Comment:
Not seeing any caller for this API except UT in the current diff. Will this be used in future PR?

Reply:
The integration test uses it to wipe out the checkpoint during cleanup.  It could be used, I could actually wrap deleteBlob around it in a future PR.